src/styles/globals.css

```diff
@@ -1,3 +1,17 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
```

Comment:
all this can be easily done with tailwind

Reply:
This was done because the padding around the logo (which must be kept) shrinks it on screens with a width lower than 300px,  mainly the galaxy fold screen, which this styling only affects. 

The breakpoint for "sm:" is a min width of 640px, which is why it proved challenging to achieve it with tailwind.